### ISSUE TITLE
Make fields clickable on product page

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-androidapp#31]
One of the killer things on Open Food Facts web version is being able to browse from product to product by clicking on the fields (eg: Poissons panés or Findus)
It would be great to leverage the new API by letting users navigate using the App

**Comment by teolemon** [comment]
Example with E301

http://world.openfoodfacts.org/additive/e301-sodium-ascorbate.json
http://world.openfoodfacts.org/additive/e301-sodium-ascorbate.xml

**Comment by teolemon** [comment]
The idea is more than having popups. It would be moving to a whole new view, much like we do on the web.

**Comment by herau** [comment]
@teolemon can you have a more detail specification ? 
If i select a field from a product, then we should display only a list view with all associated products ? or a more complicated view, like in openfoodfacts website, with sometimes a nutritional summary of all products

**Comment by teolemon** [comment]
This depends on #66

**Comment by teolemon** [comment]
in a first iteration, just all products with the criteria, and then we could add the averages…
It's a view reading the associated JSON.
So say the product has E301, on clicking it would display a new view with a list of product from http://world.openfoodfacts.org/additive/e301-sodium-ascorbate.json

**Comment by teolemon** [comment]
Having more complex view for some criteria present challenges like location (perhaps adding a map), categories (adding a table with the averages, which should be present in the JSON)…

**Comment by teolemon** [comment]
Also, we can imaging letting the user combine criteria just like on the web, but it's not a priority.

**Comment by herau** [comment]
ok i get it. i will working on the first iteration.

**Comment by teolemon** [comment]
#66 closed. 
The simple way would be to reuse the view created for search, using the various jsons.
```http://world.openfoodfacts.org/additives.json
http://world.openfoodfacts.org/additive/e301-sodium-ascorbate.json```

**Comment by Karljoones** [comment]
I am going to close this as you have split it into individual issues
